src/audio/fft/fft.c

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Amery Song <chao.song@intel.com>
+//	   Keyon Jie <yang.jie@linux.intel.com>
+
+#include <sof/common.h>
+#include "fft.h"
+#include "twiddle.h"
+
+static void icomplex32_add(struct icomplex32 *in1, struct icomplex32 *in2, struct icomplex32 *out)
+{
+	out->real = in1->real + in2->real;
+	out->image = in1->image + in2->image;
```

Comment:
Minor, is image a typo? It's imaginary, if shortened maybe imag would be better. Native speakers welcome to chime in!